capa/features/extractors/dnfile/helpers.py

```diff
@@ -143,6 +143,11 @@ def read_dotnet_method_body(pe: dnfile.dnPE, row: dnfile.mdtable.MethodDefRow) -
 
 def read_dotnet_user_string(pe: dnfile.dnPE, token: StringToken) -> Optional[str]:
     """read user string from #US stream"""
+    if pe.net.user_strings is None:
+        # #US stream may not exist - seen in obfuscated .NET
+        logger.warning("#US stream does not exist for stream index 0x%08x" % token.rid)
```

Comment:
should this be a warning or a debug message (and/or a characteristic)?

Reply:
are you asking if if `logger.debug` should be used here instead of `logger.warning`? I think this is a neat idea for a characteristic although unsure how uncommon it is for .NET to be missing the #US stream. I'll not this in #953

Reply:
Yes, on the logger verbosity (I'm leaning towards debug, but I haven't checked what other things we "warn" about).

Thanks for adding to #953.

Reply:
agreed - I've changed these to `debug`.